indico/htdocs/js/indico/modules/static_filters.js

```diff
@@ -26,6 +27,7 @@
         if (!term) {
             $contributions.show();
             setState($state, $contributions, $contributions);
+            $state.next('#total-duration').removeClass('fade');
```

Comment:
I prefer it like this, it's implied that it's right next to the `state`. Is there any major disadvantage?